### PR TITLE
feat(folder): add move upload or folder

### PR DESCRIPTION
**Pull request by Aman-Codes**
## Description

Added move upload or folder

### Changes

- Added move upload or folder

## Screenshot

image

## How to test

Go to `http://localhost:3000/organize/folders/move`

### CODE REVIEW

**Review by GMishx — APPROVED** [review]
Changes looks good. Tested, working as expected.

**Comment by Aman-Codes** [comment]
@GMishx Resolved merged conflicts

**Comment by GMishx** [comment]
There is one more for you @Aman-Codes :-)

**Comment by Aman-Codes** [comment]
@GMishx Resolved again :)

File: src/components/Widgets/Input/index.js [hunk 1]
```diff
@@ -63,8 +62,6 @@ const InputContainer = ({
           className={className && `mr-2 ${className}`}
           value={value}
           onChange={onChange}
-          multiple={multiple && multiple}
-          size={multiple && "15"}
```
Comment (GMishx) [inline src/components/Widgets/Input/index.js:67]: Are you sure about removing it? I think it was added for a large drop-down by @Shruti3004

Comment (Aman-Codes) [reply src/components/Widgets/Input/index.js:67]: I am unable to understand the meaning of `{multiple && multiple}` where `multiple` is a string according to `prop-type`.
Maybe @Shruti3004 could explain better.

Comment (Aman-Codes) [reply src/components/Widgets/Input/index.js]: Updated it

Comment (Shruti3004) [reply src/components/Widgets/Input/index.js]: Yes it is for the multiple selection of files.

**Pull request merged**